docs/develop/consistency.md

```diff
@@ -1,224 +0,0 @@
----
-sidebar_position: 5
----
-
-# Consistency
-
-TigerBeetle is designed to guard against bugs not only in its
-[own code](https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md), but at the
-boundaries, in the application code which interfaces with TigerBeetle. This is exhibited by the
-client's API design, which may be surprising (see [Retries](#retries)) when contrasted with a more
-conventional database.
-
-Strict consistency guarantees (at the database level) simplify 1) application logic and 2) error
-handling farther up the stack.
-
-## Guarantees
-
-TigerBeetle provides strict serializability
-([serializability + linearizability](http://www.bailis.org/blog/linearizability-versus-serializability/))
-to each [client session](./client-sessions.md).
```

Comment:
Would be good to move these two intro paragrpahs to the `Concepts` section, once we have it. It's important to mention "strict serializability" _before_ we get to the reference

Reply:
Agreed!